src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    /**
+     * Global variables
+     */
+    private static final String SPACE = "    ";
+    private static final String LINE = SPACE + "______________________________________________________________________";
+    private static final String END_LINE = SPACE + "======================================================================\n";
+    private static final String LOGO = "\n" +
+            "███████████████████████████████\n" +
+            "█▄─▄─▀█─█─█▄─██─▄█─▄─▄─█▄─██─▄█\n" +
+            "██─▄─▀█─▄─██─██─████─████─██─██\n" +
+            "▀▄▄▄▄▀▀▄▀▄▀▀▄▄▄▄▀▀▀▄▄▄▀▀▀▄▄▄▄▀▀";
+    private Items items;
+
+    public Ui() {
+        items = new Items();
+    }
+
+    /**
+     * returns a greeting to the user
```

Comment:
Perhaps need a full stop at the end? 🤔
And I noticed the same issue in several other places below.